rules-reviewed/camel3/camel2/xml-generic-information.windup.xml

```diff
@@ -30,7 +30,103 @@
+                    </xmlfile>
+                    <xmlfile matches="//*/c:unmarshal/c:zipFile">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`zipFile` data format was renamed to `zipfile`" effort="1"
+                      category-id="mandatory">
+                    <message>`zipFile` data format was renamed to `zipfile`
+
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_xml_dsl_migration"
+                          title="Camel 3 - Migration Guide"/>
+                    <quickfix type="REPLACE" name="camel.zipfile.dataformat.replacement.spring">
+                        <replacement>c:zipfile</replacement>
+                        <search>c:zipFile</search>
+                    </quickfix>
+                    <quickfix type="REPLACE" name="camel.dataformat.zipfile.replacement.blueprint">
+                        <replacement>b:zipfile</replacement>
+                        <search>b:zipFile</search>
+                    </quickfix>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-generic-information-00004">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:secureXML/@keyOrTrustStoreParametersId">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:secureXML/@keyOrTrustStoreParametersId">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`keyOrTrustStoreParametersId` attribute was renamed" effort="1"
+                      category-id="mandatory">
+                    <message>`keyOrTrustStoreParametersId` attribute was renamed to `keyOrTrustStoreParametersRef`
+
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_xml_dsl_migration"
+                          title="Camel 3 - Migration Guide"/>
+                    <quickfix type="REPLACE" name="camel.dataformat.securexml.replacement.spring">
+                        <replacement>c:secureXML/@keyOrTrustStoreParametersId</replacement>
+                        <search>c:secureXML/@keyOrTrustStoreParametersId</search>
+                    </quickfix>
+                    <quickfix type="REPLACE" name="camel.dataformat.securexml.replacement.blueprint">
+                        <replacement>b:secureXML/@keyOrTrustStoreParametersRef</replacement>
+                        <search>b:secureXML/@keyOrTrustStoreParametersId</search>
+                    </quickfix>
```

Comment:
The `quickfix` does text searches, therefore the search and replace should be modified.  
I would suggest to have only 1 Quickfix with this code 
```
                    <quickfix type="REPLACE" name="camel.dataformat.securexml.replacement.blueprint">
                        <replacement>keyOrTrustStoreParametersRef</replacement>
                        <search>keyOrTrustStoreParametersId</search>
                    </quickfix>
```